MANIFEST.in

```diff
@@ -9,3 +9,5 @@ recursive-include shelltests *.sh testrunner testlib
 recursive-include tests *.py
 recursive-include test-tooling resalloc* wrap
 recursive-include build_manpages/build_manpages *.py
+recursive-include resallocwebui/static/*.css
+recursive-include resallocwebui/templates/*.html
```

Comment:
This seems to be the problem ... note the pattern used in the old MANIFEST lines.